ddev/tests/cli/validate/test_metrics.py

```diff
@@ -5,7 +5,64 @@
 import os
 from pathlib import Path
 
-from datadog_checks.dev.fs import write_file
+import pytest
+
+from .conftest import _fake_repo as _base_fake_repo
+from .conftest import write_file
+
+
+def _fake_repo(tmp_path_factory, config_file, name):
+    repo = _base_fake_repo(tmp_path_factory, config_file, name)
+
+    write_file(
+        repo.path / 'metadata_integration',
+        'manifest.json',
+        """{
+          "manifest_version": "2.0.0",
+          "app_uuid": "15b15f01-b342-4001-89ac-9e92fc4f3234",
+          "app_id": "druid",
+          "display_on_public_website": true,
+          "assets": {
+            "integration": {
+              "metrics": {
+                "prefix": "metadata_integration.",
+                "check": [],
+                "metadata_path": "metadata.csv"
+              }
+            }
+          }
+        }
+""",
+    )
+
+    write_file(
+        repo.path / 'metadata_integration',
+        'metadata.csv',
+        """metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
```

Comment:
What's the extra empty line for?

Also, would it be more easy to read/modify if we use the `csv.Writer` to create the file from a list of lists/dicts?. Keeping track of the commas is something computers would beat me at any day :)

Reply:
I think it will only add extra complexity for something very simple. Here at least we have the file as it is, without any processing that might introduce a bug

Reply:
alrighty.